how-to/observability.rst

```diff
@@ -5,23 +5,8 @@ Observability
 
 In Ubuntu, it is recommended to use the
 `Canonical Observability Stack <https://documentation.ubuntu.com/observability/>`_
-to monitor your infrastructure.
+to monitor your infrastructure. For more information, see the following links:
 
-However, you can also use the classic Logging, Monitoring and Alerting (LMA)
-stack.
-
-.. toctree::
-   :titlesonly:
-
-   Set up your LMA stack <observability/set-up-your-lma-stack>
-
-If you do not need a full LMA stack, there are some other supported tools that
-can provide similar capabilities.
-
-.. toctree::
-   :titlesonly:
-
-   Install Logwatch <observability/install-logwatch>
-   Install Munin <observability/install-munin>
-   Install Nagios Core 3 <observability/install-nagios>
-   Use Nagios with Munin <observability/use-nagios-with-munin>
+- `Getting started with COS Lite <https://documentation.ubuntu.com/observability/tutorial/installation/getting-started-with-cos-lite/>`_
+- `How to integrate COS Lite with uncharmed applications <https://documentation.ubuntu.com/observability/how-to/integrating-cos-lite-with-uncharmed-applications/>`_
+- `The OpenTelemetry Collector snap <https://snapcraft.io/opentelemetry-collector>`_
```

Comment:
These links are a really helpful addition, but if we want to tempt people to use COS we should provide a bit more context around these links.

I think if we can use this space to disambiguate the difference between the COS and COS lite use cases, and say what OTC does and what use case it is for, it will help our readers to understand which of the options they should use for their situation, which will in turn help increase the adoption.